<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>728</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="Main_ui">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Products</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Product List</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>22</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="cb_out_of_stock">
            <property name="text">
             <string>View Out of stock</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pb_create">
              <property name="text">
               <string>Create</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_delete">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_update">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="8" column="1">
         <widget class="QTableView" name="tv_product_list"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Orders</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="1">
               <widget class="QPushButton" name="pb_order_update">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="pb_order_delete">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pb_order_process">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Process</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="pb_order_create">
                <property name="text">
                 <string>Create</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <family>Comic Sans MS</family>
                    <pointsize>12</pointsize>
                    <kerning>false</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>Filter Order By Name:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_order_filter"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QRadioButton" name="rb_order_no_filter">
              <property name="text">
               <string>No Filter</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_order_unprocessed">
              <property name="text">
               <string>Unprocessed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_order_processed">
              <property name="text">
               <string>Processed</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Orders</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="tv_order_view"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Order's Item</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="tv_order_items">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="actionCreated_By_Quinten"/>
     <addaction name="actionA01055681"/>
     <addaction name="actionQuinten_Leung"/>
    </widget>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionAdd_New_Product">
   <property name="text">
    <string>Create New Product</string>
   </property>
  </action>
  <action name="actionCreate_New_Order">
   <property name="text">
    <string>Create New Order</string>
   </property>
  </action>
  <action name="actionEdit_Order">
   <property name="text">
    <string>Edit Order</string>
   </property>
  </action>
  <action name="actionCreated_By_Quinten_Leung">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Created By Quinten Leung</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCreated_By_Quinten">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Created By:</string>
   </property>
  </action>
  <action name="actionA01055681">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>A01055681</string>
   </property>
  </action>
  <action name="actionQuinten_Leung">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quinten Leung</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>Main_ui</tabstop>
  <tabstop>cb_out_of_stock</tabstop>
  <tabstop>tv_product_list</tabstop>
  <tabstop>pb_create</tabstop>
  <tabstop>pb_delete</tabstop>
  <tabstop>pb_update</tabstop>
  <tabstop>le_order_filter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
